docs/pages/api-reference/operators/changelog.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Changelog
+order: 0
+status: published
+---
+### Changelog
+Operator to convert a keyed dataset into a keyless one, where the underlying delta
+frame of the dataset is presented as a changelog. All key fields are converted into
+normal fields, and an additional column is added which contains the kind (insert
+or delete) of the delta.
```

Comment:
users dont know about delta frame ( an internal detail ),

Operator to convert a keyed dataset into a CDC changelog stream. All key fields are converted into value fields, and an additional column is added, indicating the type of change (insert or delete) for the delta.

Reply:
Python users don't know about value fields either

Reply:
they do